packages/lesswrong/components/ultraFeed/FeedItemWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { registerComponent } from '../../lib/vulcan-lib/components';
+import { defineStyles, useStyles } from '../../components/hooks/useStyles';
+
+const styles = defineStyles('FeedItemWrapper', (theme: ThemeType) => ({
+  wrapper: {
+    marginBottom: 0,
+    borderBottom: '2px solid rgba(0,0,0,0.05)'
```

Comment:
should be using `theme.palette.greyAlpha(0.05)` or something, rather than an inline `rgba` (which causes a unit test failure)